ubuntu/pyenv-python-poetry.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash -e
+
+# Define these 2 variables if you want to customize the default installation,
+# then copy-paste the remainder of the script:
+# - the version of python to install
+# python_version=3.8.6
+# - the poetry version to install (if not specified, installs the last one)
+# poetry_version=1.1.5
+# - the path to the labs folder, where the python version will be activated
+# labs_path=/home/ubuntu/labs_python
+
+python_version=3.8.6
+poetry_version=1.1.5
```

Comment:
Si tu ne suis pas tes propres recos... 😝 

```suggestion
python_version=${python_version:-3.8.6}
poetry_version=${poetry_version:-1.1.5}
```

Reply:
ah, je n'avais pas présenté les choses ainsi (cf. https://github.com/Zenika/strigo-init-script-libs/pull/8/files#diff-aa2475d4f8253db9a16955859cf75ba0a5d4f2e2418db0d07390376ca47f1e48R16-R17) : mais tu as raison :smile: